lib/delayed/backend/actions.rb

```diff
@@ -30,31 +30,37 @@ def db_time_now
         #end
 
         def find_available(worker_name, limit = 5, max_run_time = Worker.max_run_time)
-          Delayed::Worker.available_priorities.each do |priority|
-            message = nil
-            begin
-              message = ironmq.queue(queue_name(priority)).get
-            rescue Exception => e
-              Delayed::Worker.logger.warn(e.message)
+          Delayed::IronMqBackend.available_priorities.each do |priority|
+            Delayed::IronMqBackend.queues.each do |queue_item|
+              message = nil
+              queue = queue_name(queue_item, priority)
+              begin
+                message = ironmq.queue(queue).get
+              rescue Exception => e
+                Delayed::IronMqBackend.logger.warn(e.message)
+              end
+              return [Delayed::Backend::Ironmq::Job.new(message)] if message
             end
-            return [Delayed::Backend::Ironmq::Job.new(message)] if message
           end
           []
         end
 
         def delete_all
-          Delayed::Worker.available_priorities.each do |priority|
+          Delayed::IronMqBackend.available_priorities.each do |priority|
             loop do
               msgs = nil
-              queue = queue_name(priority)
-              begin
-                msgs = ironmq.queue(queue).get(:n => 1000)
-              rescue Exception => e
-                Delayed::Worker.logger.warn(e.message)
+              Delayed::IronMqBackend.queues.each do |queue_item|
```

Comment:
`Delayed::IronMqBackend.queues` is `nil` by default and  this line leads to an error. am i wrong?